src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: BrowserWindowSession
+excerpt: ""
+description: "Captures sessions linked to the focus of Electron BrowserWindows."
+sidebar_order: 15
+---
+
```

Comment:
Let's add a note that this requires `4.11.0`.

Should we also talk about this affects the MainProcessSession integration?